core/tests/protractor_utils/forms.js

```diff
@@ -588,6 +588,11 @@ var toRichText = async function(text) {
  * loads more divs.
  */
 var CodeMirrorChecker = function(elem, codeMirrorPaneToScroll) {
+  var lineContentElements = elem.all(by.css('.CodeMirror-line'));
+  var lineNumberElements = elem.all(by.css('.CodeMirror-linenumber'));
+  var scrollBarElements = element.all(by.css('.CodeMirror-vscrollbar'));
+  var CodeMirrorLinenumberLocator = by.css('.CodeMirror-linenumber');
+  var CodeMirrorLinebackgroundLocator = by.css('.CodeMirror-linebackground');
```

Comment:
Please keep these camelCase:

```suggestion
  var codeMirrorLineNumberLocator = by.css('.CodeMirror-linenumber');
  var codeMirrorLineBackgroundLocator = by.css('.CodeMirror-linebackground');
```

Reply:
Fixed!